docs/dev/google-drive.md

```
@@ -56,35 +57,41 @@ GOOGLE_OAUTH_CLIENT_ID=<your-client-id>.apps.googleusercontent.com
+The script connects to a single persistent `workspace-mcp` process (important — each
+service auth call must happen in the same process so the OAuth state survives the
+callback), opens your browser once per service, and waits for you to complete the login.
 
-# Run the MCP server — it will print an OAuth URL
-GOOGLE_OAUTH_CLIENT_ID=<...> GOOGLE_OAUTH_CLIENT_SECRET=<...> uvx workspace-mcp
+Log in as **Curia's Gmail account** each time.
 
-# In the browser: open the printed URL, log in as Curia's Gmail, and approve access.
-# Tokens are saved to ~/.workspace-mcp/cli-tokens/.
+After all five services are authenticated, tokens are saved to:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)